app/views/provider_interface/conditions/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :browser_title, 'Confirm conditions' %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_path(@application_choice.id)) %>
+
+<%= form_with model: @conditions_form,
+      url: provider_interface_application_choice_confirm_update_conditions_path(@application_choice.id),
+      method: :patch do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-xl">
+    Confirm if the candidate has met all conditions
+  </h1>
+
+  <%= render SummaryListComponent, rows: [
+    { key: 'Candidate name', value: @application_choice.application_form.full_name },
+    { key: 'Course', value: @application_choice.course.name_and_code },
+    { key: 'Preferred location', value: @application_choice.site.name },
+    { key: 'Provider', value: @application_choice.course.provider.name_and_code },
+  ] %>
+
+  <h2 class="govuk-heading-l govuk-!-margin-top-8">
+    Conditions
+  </h2>
+
+  <% condition_rows = @application_choice.offer['conditions'].map { |condition| { value: condition } } %>
+  <% if condition_rows.empty? %>
+    <%= render SummaryListComponent, rows: [{ value: 'No conditions have been specified' }] %>
+  <% else %>
+    <%= render SummaryListComponent, rows: condition_rows %>
+  <% end %>
```

Comment:
Could we re-use the code from the offer confirmation page here?

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/views/provider_interface/decisions/confirm_offer.html.erb#L19-L35

That means we won't have to change the summary list component.

Reply:
Problem is this code doesn't match the design Adam has given me. Allowing SummaryListComponent to list rows without keys is a minimal and useful change, no?